libs/platform/experience/entity-link/entity-link.component.ts

```diff
@@ -0,0 +1,50 @@
+import { EntityService } from '@spryker-oryx/core';
+import { resolve } from '@spryker-oryx/di';
+import { ContentMixin } from '@spryker-oryx/experience';
+import { computed, hydrate } from '@spryker-oryx/utilities';
+import { LitElement, TemplateResult } from 'lit';
+import { html } from 'lit/static-html.js';
+import { catchError, of } from 'rxjs';
+import { EntityLinkOptions } from './entity-link.model';
+
+@hydrate()
+export class EntityLinkComponent extends ContentMixin<EntityLinkOptions>(
+  LitElement
+) {
+  protected entityService = resolve(EntityService);
+
+  protected $data = computed<string | undefined>(() => {
+    const { entity: type, field } = this.$options();
+    return this.entityService
+      .getField<string>({ element: this, type, field })
+      .pipe(catchError(() => of()));
```

Comment:
Maybe better to perform error handling on service level to avoid rxjs in components?

Reply:
What about this one? I saw the same case inside Stan's PR, he replied that handling errors inside services can limit us in covering some specific cases on place, but I'm not sure that this is also a right way, should we move error handling to the components?

Reply:
Let's leave it here for now. We're not sure if we want to hide it in service layer as of now.